static/js/tippyjs.js

```diff
@@ -482,3 +482,22 @@ export function initialize() {
         appendTo: () => document.body,
     });
 }
+
+export function show_copied_confirmation($copy_button) {
+    // Display a tooltip to notify the user the message or code was copied.
+    const instance = tippy($copy_button, {
+        placement: "top",
+        appendTo: () => document.body,
+        onUntrigger() {
+            remove_instance();
+        },
+    });
+    instance.setContent("Copied!");
+    instance.show();
+    function remove_instance() {
+        if (!instance.state.isDestroyed) {
+            instance.destroy();
+        }
+    }
+    setTimeout(remove_instance, 3000);
```

Comment:
Looking at this, we can simply change the text of the existing `tooltip` attached to the `$copy_button`. Also, remember to mark `Copied!` text for translation.

Reply:
That doesn't seem to work on ```copy code``` button since It would update it back to ```copy code``` text the moment it is changed and there doesn't seem to be a way to stop it.

Reply:
I pushed the PR to implement it, can you verify if it is working correctly?

Reply:
I tried the same yesterday(didn't work), also checked your PR but its not working for copy code button.
![chrome-capture-2023-1-22](https://user-images.githubusercontent.com/99543661/220552478-09124fe1-d32a-49e8-818c-2bef5c741665.gif)
Its showing somewhat visible in GIF but for the naked eyes, the ```copied!``` message is barely visible.
It does work sometime but mostly the ```copied!``` message doesn't show.

Reply:
oh! it does work if you just tap on the button, but doesn't if you hard press the button. Anyway, that's not acceptable, so you can push your original changes.